src/odemis/util/transform.py

```diff
@@ -193,6 +193,70 @@ def _optimal_rotation(x, y):
     return numpy.dot(V.T, U.T)
 
 
+def to_physical_space(ji, shape, pixel_size=None):
+    """
+    Converts an image index into a coordinate in physical space.
+
+    Images are displayed on a computer screen with the origin in the top-left
+    corner. The column-index `i` is used as the primary axis and the row-index
+    `j` is used as the secondary axis. This represents a left-handed coordinate
+    system. Physical coordinates are typically described in a right-handed
+    coordinate system.
+
+    Calculations should be done preferably in the right-handed physical
+    coordinate system. Calculations using mixed left-handed and right-handed
+    coordinates should be avoided.
+
+    `to_physical_space` converts an image index into a coordinate in physical
+    space. The sign of the row-index is changed such that is increasing in the
+    upward direction, the origin is moved to the center of the image, and the
+    `x`-axis is made the primary axis.
+
+
+    Parameters
+    ----------
+    ji : tuple, list of tuples, ndarray
+        Pixel index, or list of indices, into a 2-dimensional array.
+    shape : tuple
```

Comment:
I still think the order in shape should be also explicitly defined to avoid any confusion in future, even though it might look obvious. It also shows that the author of the code has thought about it.
`First entry is the row-index and the second entry is the column-index.`